lib/settings/pages/settings_page.dart

```diff
@@ -45,6 +51,101 @@ class SettingsPage extends StatelessWidget {
             toolbarHeight: 70.0,
             pinned: true,
           ),
+          SliverToBoxAdapter(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 18),
+              child: SearchAnchor(
```

Comment:
You can use SearchAnchor.bar to make it more Material looking (e.g., proper padding values, etc). This also fixes the issue where it looks like the SearchAnchor is typeable!

https://api.flutter.dev/flutter/material/SearchAnchor/SearchAnchor.bar.html